### DISCUSSION TITLE
Receiving error banner when trying to add "Base button" component in Figma

### DISCUSSION THREAD

**Discussion by slouie2** [twilio-labs/paste#1325] [General] [answered]
I can add all the other components from the Paste Components library fine, but  the "Base button" is giving me trouble. Any ideas on what's  going on?

<img width="649" alt="Screen Shot 2021-03-26 at 3 42 53 PM" src="https://user-images.githubusercontent.com/74426389/112691279-158a2b00-8e4b-11eb-9205-a2c00e84d420.png">

**Comment by corinne-nagel** [comment]
Hey @slouie2 ! "Base button" is part of the Figma component structure and shouldn't be showing up in the library, so not sure what's going on there 🤔 . Can you share what you were doing when you tried adding this and what you were trying to accomplish?

**Reply by slouie2** [reply]
Hi @corinne-nagel, I was trying to add the "Base button" to my design mockups. I tried clicking on/dragging it from the Assets panel but nothing worked. I've found a workaround for it by adding "Button"/the secondary variant to my mockups and then changing it's variant to "Primary" (but ideally I shouldn't have to  do that).

**Reply by serifluous** [reply]
@slouie2 Corinne's right in that if "Base button" is showing up in the library for you, that's a bug on our end.

Would you mind giving me (sali) edit access to the file so I can see what's going on with the linked library?

fwiw, the way to place the primary button in Figma is the way you ended up doing it! We made the secondary button the default because it should be the most frequently used variant.

**Reply by slouie2** [reply]
Hey @serifluous I just sent you an invite link to access the file

**Comment by serifluous** [comment, marked as answer]
Rounding out this Discussion after following up in Figma:

- The reason "Button base" is most likely showing up is because the Paste Button component was detached (which exposed "Button base") and then the Button was made a component again (local to the file).
- I recommended either:
    - Removing the local component from the file
    - Avoiding/ignoring "Button base" and using the Paste Button component only.
